lms/templates/instructor/instructor_dashboard_2/executive_summary.html

```diff
@@ -0,0 +1,135 @@
+<%! from django.utils.translation import ugettext as _ %>
+<!DOCTYPE HTML>
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<title>Edx Executive Summary</title>
```

Comment:
Executive Summary

Note: If we have to keep "edX" at the beginning, capitalize it as "EdX" (so the title would be "EdX Executive Summary").